### PR TITLE
feat: add PrevN/NextN schedule introspection to web API

### LINKED ISSUE #493: feat: add PrevN/NextN schedule introspection to web dashboard
## Summary

go-cron v0.12.0 added `PrevN` schedule introspection ([go-cron#328](https://github.com/netresearch/go-cron/pull/328)), complementing the existing `NextN` API. Currently, ofelia's web UI only shows the schedule string without calculated previous or upcoming execution times.

## Benefits

- Enhanced web dashboard with "next N runs" and "last N schedule windows"
- Better visibility into job scheduling patterns
- Helps users verify their cron expressions work as expected

## Implementation Notes

- Add `NextRuns []time.Time` and `PrevRuns []time.Time` fields to `apiJob` struct (web/server.go)
- Use `schedule.NextN(now, 5)` and `schedule.PrevN(now, 5)` on each entry's schedule
- Expose via `/api/jobs` response
- Frontend can display these in a tooltip or expandable section

## Acceptance Criteria

- [ ] API returns next 5 and previous 5 schedule times per job
- [ ] Web UI displays schedule introspection data
- [ ] Handles edge cases (triggered-only jobs, paused jobs)
- [ ] Tests for API response format

### PR DESCRIPTION
**Pull request by CybotTM**
## Summary
- Add `NextRuns` and `PrevRuns` arrays (5 entries each) to the `/api/jobs` response using go-cron v0.12.0's `NextN`/`PrevN` introspection APIs
- Add `EntryByName` method to `Scheduler` for looking up cron entries by job name with nil-safe guard
- Disabled, removed, and triggered jobs correctly return empty run arrays

## Test Plan
- [x] `TestJobsEndpointNextPrevRuns` - verifies next/prev runs populated
- [x] `TestJobsEndpointTriggeredJobEmptyRuns` - triggered jobs have empty runs
- [x] `TestDisabledJobsEndpointEmptyRuns` - disabled jobs have empty runs
- [x] `TestRemovedJobsEndpointEmptyRuns` - removed jobs have empty runs
- [x] `TestJobsEndpointNextPrevRunsJSONFormat` - JSON format validation
- [x] All existing tests pass
- [x] `golangci-lint` clean (0 issues)

Closes #493

### CODE REVIEW

File: web/server.go [hunk 1]
```diff
@@ -193,6 +194,8 @@ type apiJob struct {
 	Command  string          `json:"command"`
 	Running  bool            `json:"running"`
 	LastRun  *apiExecution   `json:"lastRun,omitempty"`
+	NextRuns []time.Time     `json:"nextRuns"`
+	PrevRuns []time.Time     `json:"prevRuns"`
```
[the remark this answers is not in the document]
Comment (CybotTM) [reply web/server.go]: Added `nextRuns` and `prevRuns` array fields to the Job schema in the OpenAPI spec. Fixed in a0174bf.
